5-AccessControl/2-call-api-groups/AppCreationScripts/sample.json

```
@@ -1,61 +1,64 @@
 {
   "Sample": {
-    "Title": "React single-page application calling a protected Node.js & Express web API using Security Groups to implement Role-Based Access Control",
-    "Level": 300,
-    "Client": "React SPA",
-    "Service": "Node.js & Express web API",
-    "RepositoryUrl": "ms-identity-javascript-react-tutorial",
-    "Endpoint": "AAD v2.0"
+      "Title": "React single-page application calling a protected Node.js & Express web API using Security Groups to implement Role-Based Access Control",
+      "Level": 300,
+      "Client": "React SPA",
+      "Service": "Node.js & Express web API",
+      "RepositoryUrl": "ms-identity-javascript-react-tutorial",
+      "Endpoint": "AAD v2.0",
+      "platform": "javascript",
+      "Languages": ["javascript"],
+      "Description": "React single-page application calling a protected Node.js & Express web API using Security Groups to implement Role-Based Access Control",
+      "products": ["azure-active-directory", "ms-graph", "msal-js", "msal-react", "msal-node"]
   },
   "AADApps": [
     {
-      "Id": "service",
-      "Name": "msal-node-api",
-      "Kind": "WebApi",
-      "Audience": "AzureADMyOrg",
-      "GroupMembershipClaims": "SecurityGroup",
-      "PasswordCredentials": "Auto",
-      "HomePage": "http://localhost:5000/api",
-      "RequiredResourcesAccess": [
-        {
-          "Resource": "Microsoft Graph",
-          "DelegatedPermissions": [ "User.Read", "GroupMember.Read.All" ]
-        }
-      ],
-      "ManualSteps": [
-        { "Comment":  "Navigate to the portal and set the 'acceptedTokenVersion' to '2' in the application manifest" },
-        { "Comment": "This script has created a group named GroupAdmin for you. On Azure portal, assign some users to it, and configure your ID and Access token to emit GroupID in your app registration." },
-        { "Comment": "This script has created a group named GroupMember for you. On Azure portal, assign some users to it, and configure your ID and Access token to emit GroupID in your app registration." }
-      ]      
-    },
-    {
-      "Id": "client",
-      "Name": "msal-react-spa",
-      "Kind": "SinglePageApplication",
-      "Audience": "AzureADMyOrg",
-      "HomePage": "http://localhost:3000/",
-      "ReplyUrls": "http://localhost:3000/",
-      "GroupMembershipClaims": "SecurityGroup",
-      "RequiredResourcesAccess": [
-        {
-          "Resource": "service",
-          "DelegatedPermissions": [ "access_as_user" ]
+        "Id": "client",
+        "Name": "msal-react-app",
+        "Kind": "SinglePageApplication",
+        "Audience": "AzureADMyOrg",
+        "HomePage": "http://localhost:3000/",
+        "ReplyUrls": "http://localhost:3000/ , http://localhost:3000/redirect.html",
+        "GroupMembershipClaims": "SecurityGroup",
+        "PasswordCredentials": "Auto",
```

```suggestion
        "PasswordCredentials": "Auto",
```

This line should've generated a step in the README for using certificates, but I don't see it in the README